bridgepoint/schema.py

```diff
@@ -2245,15 +2307,15 @@
 CREATE ROP REF_ID R501 FROM MC SM_STATE (SM_ID) TO 1 SM_SM (SM_ID);
 CREATE ROP REF_ID R502 FROM MC SM_EVT (SM_ID) TO 1 SM_SM (SM_ID);
 CREATE ROP REF_ID R503 FROM MC SM_SEME (SM_ID, SMstt_ID) TO 1 SM_STATE (SM_ID, SMstt_ID);
-CREATE ROP REF_ID R503 FROM MC SM_SEME (SM_ID, SMevt_ID) TO 1 SM_SEVT (SM_ID, SMevt_ID);
-CREATE ROP REF_ID R504 FROM 1C SM_EIGN (SM_ID, SMevt_ID, SMstt_ID) TO 1 SM_SEME (SM_ID, SMevt_ID, SMstt_ID);
-CREATE ROP REF_ID R504 FROM 1C SM_CH (SM_ID, SMevt_ID, SMstt_ID) TO 1 SM_SEME (SM_ID, SMevt_ID, SMstt_ID);
-CREATE ROP REF_ID R504 FROM 1C SM_NSTXN (SM_ID, SMevt_ID, SMstt_ID) TO 1 SM_SEME (SM_ID, SMevt_ID, SMstt_ID);
+CREATE ROP REF_ID R503 FROM MC SM_SEME (SMevt_ID, SM_ID) TO 1 SM_SEVT (SMevt_ID, SM_ID);
```

Comment:
Hmmm, have the order of these identifiers changed? I've been out of the loop for a while so not sure what changed in the metamodel

Reply:
not sure... i would not have expected the order to change. 
<img width="654" alt="image" src="https://user-images.githubusercontent.com/2875522/59869425-4a597c00-9361-11e9-8410-b516ca06590a.png">

Reply:
Actually, now when I think about it, I believe the order is irrelevant here.
Links are created by batch-relate, which uses attribute names when searching for instances.
Sorry for the false alarm :)